portfolio/src/main/webapp/script.js

```diff
@@ -93,13 +93,18 @@ function questions() {
     }
 }
 
+/** Converts a string into a float and checks if it is an int. */
+function isInteger(input) {
+    return Number.isInteger(Number.parseFloat(input));
+}
+
 /**
  * Empties the current comment history and reloads it with a new 
  * number of comments to load from num-comments textbox
  */
 function updateComments() {
     var numComments = document.getElementById("num-comments").value;
-    if(numComments < 1 || !Number.isInteger(numComments)) {
+    if(numComments < 1 || !isInteger(numComments)) {
```

Comment:
Notice that when checking whether numComments < 1, you want that comparison to be a numeric comparison, so it would be better to convert to a number before doing the comparison, otherwise you're relying on default conversion rules.